Lib/datetime.py

```diff
@@ -10,12 +10,15 @@
 
 import time as _time
 import math as _math
+from decimal import Decimal as _Decimal
+
```

Comment:
While I recognize that the spacing here isn't quite PEP8, you're making a *lot* of incidental whitespace changes in code well away from what you're touching. New commits don't need to, and shouldn't, mingle a ton of fairly minor style fixes with the real changes; it makes the patch needlessly long.